src/selectors/user.js

```diff
@@ -1,11 +1,23 @@
-// @flow
-import type {AppState, User} from '../types';
-
-export const getUser = (state: AppState): User =>
+/**
+ * 
+ * @param {AppState} state 
```

Comment:
Should probably replace with `object` or something like that, since AppState is no longer imported.

Reply:
fixed